subsys/bluetooth/controller/Kconfig

```diff
@@ -365,7 +365,7 @@ config BT_CTLR_CHAN_SEL_2
 config BT_CTLR_ADV_EXT
 	bool "LE Advertising Extensions" if !BT_LL_SW_SPLIT
 	depends on BT_CTLR_ADV_EXT_SUPPORT
-	select BT_CTLR_SCAN_REQ_NOTIFY if BT_LL_SW_SPLIT
+	select BT_CTLR_SCAN_REQ_NOTIFY if BT_LL_SW_SPLIT && BT_BROADCASTER
```

Comment:
Shouldn't this be ` || BT_OBSERVER`? we have extended commands for the scanner as well.

Reply:
No. `BT_CTLR_SCAN_REQ_NOTIFY` is code in Broadcaster that is to generate the Scan Request received notification.

Reply:
AFAIK `BT_CTLR_SCAN_REQ_NOTIFY` is for the broadcaster, not the observer

Reply:
Haha, forget what I said, I didn't read it properly. Thought it was depends for BT_CTLR_ADV_EXT